api/src/main/java/io/opencensus/metrics/MetricRegistry.java

```diff
@@ -78,33 +53,16 @@ static MetricRegistry newNoopMetricRegistry() {
   private static final class NoopMetricRegistry extends MetricRegistry {
 
     @Override
-    public <T> void addLongGauge(
-        String name,
-        String description,
-        String unit,
-        LinkedHashMap<LabelKey, LabelValue> labels,
-        T obj,
-        ToLongFunction<T> function) {
-      Utils.checkNotNull(name, "name");
-      Utils.checkNotNull(description, "description");
-      Utils.checkNotNull(unit, "unit");
-      Utils.checkNotNull(labels, "labels");
-      Utils.checkNotNull(function, "function");
-    }
-
-    @Override
-    public <T> void addDoubleGauge(
-        String name,
-        String description,
-        String unit,
-        LinkedHashMap<LabelKey, LabelValue> labels,
-        T obj,
-        ToDoubleFunction<T> function) {
-      Utils.checkNotNull(name, "name");
-      Utils.checkNotNull(description, "description");
-      Utils.checkNotNull(unit, "unit");
-      Utils.checkNotNull(labels, "labels");
-      Utils.checkNotNull(function, "function");
+    public LongGauge addLongGauge(
+        String name, String description, String unit, List<LabelKey> labelKeys) {
+      Utils.checkListElementNotNull(
+          Utils.checkNotNull(labelKeys, "labelKeys should not be null."),
```

Comment:
We have a convention that for null check we use the message as the name of the variable. so this should be: `Utils.checkNotNull(labelKeys, "labelKeys")`